srcpkgs/66/template

```diff
@@ -0,0 +1,18 @@
+# Template file for '66'
+pkgname=66
+version=v0.1.0.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="s6-rc s6-portable-utils"
+makedepends="oblibs-devel skalibs-devel execline-devel s6-devel"
+depends="s6-rc s6-portable-utils"
+short_desc="Helpers tools around S6 supervision suite"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="ISC"
+homepage="http://web.obarun.org/software/"
+distfiles="https://framagit.org/Obarun/${pkgname}/-/archive/${version}/${pkgname}-${version}.tar.bz2"
```

Comment:
please write out the pkgname= as it is shorter and it is easier to grep this string.